engine/engine/src/engine.cpp

```diff
@@ -1542,6 +1544,7 @@ namespace dmEngine
                     return;
                 }
 
+                dmInput::Update(engine->m_InputContext);
```

Comment:
I still don't fully understand the need to have the separation between the input/binding contexts, but to better handle any new connections I added an Update() function.